bot/exts/evergreen/trivia_quiz.py

```diff
@@ -578,7 +643,7 @@ async def send_answer(
             description="",
         )
 
-        if info is not None:
+        if info:
```

Comment:
Oh I see, it's because of https://github.com/python-discord/sir-lancebot/pull/618#issuecomment-857994321
I suppose you could add a comment as to why, or just use `null` instead of an empty string in the questions.